nemo_text_processing/g2p/data/data_utils.py

```diff
@@ -146,6 +146,10 @@ def ipa_word_tokenize(text):
     return _word_tokenize(words)
 
 
+def ipa_text_preprocessing(text):
+    return text.lower()
```

Comment:
[Discussion] Do we need such strong requirements for any languages using IPA? Letter cases may lead to different pronunciations for German.

Reply:
This is just for tokenization, right?

Either way, I'd err on the side of having all graphemes be uppercased due to IPA symbol collisions with letters (e.g. "j").

Reply:
It would make sense to leave capitalization in grapheme only training, but for training with phonemes/IPA it would not be worth the complexity unless you have a G2P system that can infer different pronunciations based on the capitalization.

Or if we do want to use it, we may need to rethink the convention of grapheme words being represented as all UPPERCASE, both in the tokenizer and in the pronunciation dictionaries.

Reply:
for German dict, I chose to keep the uppercase and the word surface mixes with upper and lower cases.

Reply:
we can simplify that with the current solution now, and maybe we could iterate if we have more datapoints.